### ISSUE TITLE
list of helpful in plain easily understandable english 

### ISSUE THREAD

**Issue by SolveAllProblemsRightNow** [nomacs/nomacs#301]
seeing that this doesnt do the 2 basic things i was looking for in a main/primary image viewer,

maybe i could find some other use & value with this software,

where could we find a list of helpful things this does from most helpful to least shown in plain english that most users & most non-technical ppl would understand?

- https://github.com/nomacs/nomacs/issues/296
- https://github.com/nomacs/nomacs/issues/295

**Comment by diemmarkus** [comment]
https://nomacs.org/features/

**Comment by SolveAllProblemsRightNow** [comment]
this isnt clear

& it's not ordered/sorted from most common uses to least

'zoomable grid preview of thumbnails' should be 'you can zoom the previews pics' for example
